client/src/react-social.d.ts

```diff
@@ -5,8 +5,15 @@ module "react-social" {
   type BaseProps = {
     onClick: () => void;
     className: string;
+    /**
+     * For EmailButton components, this prop becomes the BODY of the email itself,
+     * so it can be just a url or a larger string of text.
+     */
     url: string;
     windowOptions?: string[];
+    /**
+     * For EmailButton components, this prop becomes the SUBJECT of the email.
+     */
     message?: string;
```

Comment:
@toolness so frustratingly, these `url` and `message` properties have different uses depending on what component is interacting with them. For tweets, `message` is the body of the tweet with `url` being a trailing url at the end of the tweet. However, for emails, `message` is actually the subject of the email when `url` corresponds to the body of the email, and therefore can be more than just a simple URL. 

I tried adding comments here and elsewhere to make this distinction more clear.

Reply:
Oh weird!  Thanks for clarifying this in the code!